src/main/java/com/lalabrand/ecommerce/user/wishlist/WishlistController.java

```diff
@@ -24,9 +25,9 @@ public WishlistController(WishlistService wishlistService, CommonUtils commonUti
     @QueryMapping(name = "wishlist")
     @PreAuthorize("hasAuthority('USER')")
     public WishlistDTO findWishlistForCurrentUser() {
-        Optional<User> user = commonUtils.getCurrentUser();
-        if (user.isPresent()) {
-            return wishlistService.findWishlistByUserId(user.get().getId()).orElse(null);
+        UserDetailsImpl user = commonUtils.getCurrentUser();
+        if (user != null) {
```

Comment:
@denomelchenko Same as before.
`user == null` is a bad scenario that must be checked. You can assume that the user is not null for a success scenario.

Reply:
Fixed